experimental/split_grpc/client/src/main.rs

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#[macro_use]
+extern crate log;
```

Comment:
I don't think this is necessary in 2018 edition of Rust. You should be able to just use `log::info!(...)`, or add `use log::info` if you prefer.

Reply:
Will include as part of follow-up PR.